plugins/ecue.js

```diff
@@ -219,7 +219,7 @@ function exportHandleChannel(fixture, mode, dmxCount, viewPosCount, fineChannels
   dmxByte1++;
 
   if (!('name' in channel)) {
-    channel.name = chKey;
+    channel.name = mode.channels[dmxCount];
```

Comment:
I agree, but this issue was already present before my changes in this PR and therefore, I'd suggest waiting until e:cue gets updated to use the model.